app/models/institution.rb

```diff
@@ -154,10 +152,6 @@ class Institution < ApplicationRecord
     }
   }.freeze
 
-  validates :facility_code, uniqueness: true, presence: true
-  validates :version, :institution, :country, presence: true
-  validates :institution_type_name, inclusion: { in: TYPES }
```

Comment:
These were moved to the originating CSVs since institution objects are built using SQL only and not ActiveRecord in https://github.com/department-of-veterans-affairs/gibct-data-service/blob/master/app/models/institution_builder.rb
See https://github.com/department-of-veterans-affairs/gibct-data-service/pull/516/commits/4dfe21f0a9ffe80759d14e2ffe83a9e81656f889 and https://github.com/department-of-veterans-affairs/gibct-data-service/pull/516/commits/1ab62273bdd4e8fbb7a41afb177ff8d3072ce6e5